Dz2

nekdfl opened the pull request:
home work2

nekdfl committed:
lesson 1

nekdfl committed:
lesson 2 task1

nekdfl committed:
lesson 2 task1
Вначале писал структуру, потом код. Вернул pass

nekdfl committed:
lesson 2 task2

nekdfl committed:
lesson 2 task3

nekdfl committed:
lesson 2 task4
исправил ошибку в функции input_number, вводе не правильного занчения.
добавил блок try except в функции main task2, task3, task4

nekdfl committed:
lesson 2 task5

nekdfl committed:
lesson 2 task5
Добавил печать линий таблицы
Исправил ошибку - печаталось на 1 элемент в строке больше чем задано в параметрах

nekdfl committed:
lesson 2 task6

nekdfl committed:
lesson 2 task7
Получилось совсем неудачное решение - куча дополнительных условий.

DmitryChitalov commented on Урок 2. Практическое задание/task_1.py:
```diff
@@ -28,3 +28,75 @@
+
+def ask_number(message, invalid_number=[], invalid_number_message=""):
+    pass
+    invalid_number = list(invalid_number)
```
для чего здесь pass
он здесь не нужен

DmitryChitalov commented on Урок 2. Практическое задание/task_1.py:
```diff
@@ -28,3 +28,75 @@
+
+def do_calculate(arg1, arg2, operation):
+    pass
+    res = 0
```
аналогично

nekdfl committed:
lesson 3 task1

nekdfl committed:
lesson 3 task2

nekdfl committed:
lesson 3 task3
Так и не понял до конца, что такое подстрока. Точнее как ее получить. Надеюсь подстроки выделил правильно

nekdfl committed:
lesson 3 task4
